src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -43,19 +70,30 @@ export enum PipelineRunEventTypeLabel {
 const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunListRowProps>> = ({
   obj,
   showVulnerabilities,
+  showWorkspace,
+  showTestResult,
+  showSnapshot,
+  showComponent,
+  showTrigger,
+  showReference,
   customData,
 }) => {
+  const namespace = useNamespace();
   const capitalize = (label: string) => {
     return label && label.charAt(0).toUpperCase() + label.slice(1);
   };
+  const {
+    releaseName = '',
+    releasePlan = { spec: { application: '' } },
+    release = { spec: { snapshot: '' } },
```

Comment:
nit: do we really need to have the defaults values for both `releasePlan` and `release`? 🤔